components/decimal/src/provider.rs

```diff
@@ -229,7 +229,11 @@ pub struct DecimalSymbolsV2<'data> {
 /// including in SemVer minor releases. While the serde representation of data structs is guaranteed
 /// to be stable, their Rust representation might not be. Use with caution.
 /// </div>
-#[icu_provider::data_struct(DecimalDigitsV1Marker = "decimal/digits@1")]
+#[icu_provider::data_struct(marker(
+    DecimalDigitsV1Marker,
+    "decimal/digits@1",
+    attributes_domain = "numsys"
```

Comment:
Suggestion: we should probably name the domain "u-nu" or "nu" or "numbering_system" or "numberingSystem" or something. "numsys" is not very jargony.